spec/latest/map-spec.md

```diff
@@ -220,7 +220,9 @@ set {
 
 OperationCall: `call` Iteration? OperationName OperationArguments Condition? { OperationCallSlot* }
 
-OperationArguments : ( Argument[comma]* )
+OperationArguments : ( ArgumentList* )
+
+ArgumentList: ArgumentList `,`? Argument
```

Comment:
I don't think this is valid @zdne, it's forever recursive.

In the link you posted (https://spec-md.com/#sec-Quantifiers.Use-with-Non-Terminals) they present have two branches, and the second one is always terminal (never recurses). Also the optional `|` is there to allow the "TypeScript" syntax with a leading `|`:
```ts
type MyUnion =
  | Foo
  | Bar;
```

Our rule never terminates. I suggest with go with your second suggestion, with an added optional trailing comma in `ArgumentList` (which is allowed in current parser):
```
OperationArguments : ( ArgumentList* )

ArgumentList : Argument ArgumentListContinued* `,`?

ArgumentListContinued : , Argument
```

Reply:
Ok!

Reply:
Let's fix it and focus on other stuff :/